pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -38,6 +40,7 @@ func (t *Task) createDestinationPVCs() error {
 		}
 
 		newSpec := srcPVC.Spec
+		srcPVC.Labels["owner"] = "directvolumemigration"
```

Comment:
@dymurray @sseago Is adding this label fine? This is to get PVCList to check if each PVC included in the migration is mounted or not. I am open to suggestion for other label or annotation, I just need a selector.

Reply:
@JaydipGabani Please sync up with @shubham-pampattiwar who is working on https://github.com/konveyor/mig-controller/issues/840

I've asked him to update the code to set a common label for all components in DVM so that we aren't using strings everywhere. I think this will solve your issue.